PATCH-P0-FIX-SUSE: Fix log file path for logrotate

In SUSE we've got MySQL log in different directory. It's located in
/var/log/mysql by default. It also adds some extra error message.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: support-files/mariadb.logrotate.sh
===================================================================
--- support-files/mariadb.logrotate.sh.orig
+++ support-files/mariadb.logrotate.sh
@@ -32,6 +32,14 @@
     then
       @bindir@/mariadb-admin $EXTRAPARAM --local flush-error-log \
         flush-engine-log flush-general-log flush-slow-log
+	   ret=$?
+	   if test $ret -ne 0
+	   then
+		echo "/etc/logrotate.d/mariadb failed, probably because" >&2
+		echo "the root acount is protected by password." >&2
+		echo "See comments in /etc/logrotate.d/mariadb on how to fix this" >&2
+		exit $ret
+	   fi
     fi
   endscript
 }
